### PR TITLE
Update to use app ws auth

### PR DESCRIPTION
**Pull request by ThetaSinner**

### CODE REVIEW

File: ts/test/local/scenario.ts [hunk 1]
```diff
@@ -170,25 +170,31 @@ test("Local Scenario - Conductor maintains data after shutdown and restart", asy
   const readContent = await bobCaller<typeof content>("read", createEntryHash);
   t.equal(readContent, content);
 
+  console.log("Ready to shutdown");
```
Comment (jost-s) [inline ts/test/local/scenario.ts:173]:
```suggestion
```

File: ts/test/local/scenario.ts [hunk 2]
```diff
@@ -170,25 +170,31 @@ test("Local Scenario - Conductor maintains data after shutdown and restart", asy
...
-      cell_id: bob.cells[0].cell_id,
-      zome_name: TEST_ZOME_NAME,
-      fn_name: "read",
-      payload: createEntryHash,
-    }
+  const [appInterfaceInfo] = await bob.conductor.adminWs().listAppInterfaces();
+  const issuedBob = await bob.conductor
+    .adminWs()
+    .issueAppAuthenticationToken({ installed_app_id: bob.appId });
+  console.log("Connect new client");
```
Comment (jost-s) [inline ts/test/local/scenario.ts:183]:
```suggestion
```

File: ts/test/local/scenario.ts [hunk 3]
```diff
@@ -170,25 +170,31 @@ test("Local Scenario - Conductor maintains data after shutdown and restart", asy
...
+  bob.appWs = await bob.conductor.connectAppWs(
+    issuedBob.token,
+    appInterfaceInfo.port
   );
+  console.log("Connected");
```
Comment (jost-s) [inline ts/test/local/scenario.ts:188]:
```suggestion
```

File: ts/test/local/scenario.ts [hunk 4]
```diff
@@ -170,25 +170,31 @@ test("Local Scenario - Conductor maintains data after shutdown and restart", asy
...
+  const readContentAfterRestart: typeof content = await bob.appWs.callZome({
+    cell_id: bob.cells[0].cell_id,
+    zome_name: TEST_ZOME_NAME,
+    fn_name: "read",
+    payload: createEntryHash,
+  });
   t.equal(readContentAfterRestart, content);
 
+  console.log("Done");
```
Comment (jost-s) [inline ts/test/local/scenario.ts:197]:
```suggestion
```

File: ts/src/trycp/conductor/scenario.ts [hunk 5]
```diff
@@ -156,22 +156,32 @@ export class TryCpScenario {
...
-                          conductor,
-                          appAgentWs,
-                          ...agentApp,
-                        })
+                appInfos.map(async (appInfo) => {
+                  const { port } = await adminWs.attachAppInterface();
+                  const issued = await adminWs.issueAppAuthenticationToken({
+                    installed_app_id: appInfo.installed_app_id,
+                  });
+                  // This doesn't make a lot of sense... but we are asking the trycp server to create a connection,
```
Comment (jost-s) [inline ts/src/trycp/conductor/scenario.ts:165]: Yeah, I guess a TryCP socket relay conundrum, like the attach and connect interface.

**Review by jost-s — APPROVED** [review]

File: ts/src/trycp/conductor/scenario.ts (hunk 5 above)
Comment (ThetaSinner) [reply ts/src/trycp/conductor/scenario.ts:165]: Exactly that, it's not faithfully implementing the interface but actually doing something subtly different that is now very obvious from outside :/

**Pull request merged**